headshots.py

```diff
@@ -1,19 +1,26 @@
-import cv2
+'''
+This code takes pictures of the user its recommended to take multiple per person
+'''
 import os
+import cv2
+import functions
 
 DATASET_FOLDER = "dataset"
-NAME = 'donkeykong' #replace with your name
+NAME = 'user2' #replace with your name
```

Comment:
If it's going to be needed at every usage, perhaps it would be a good idea to turn this into a normal variable (lower case) and ask input from the user on the console.